specification/sql/resource-manager/Microsoft.Sql/preview/2020-02-02-preview/DevOpsAudit.json

```diff
@@ -0,0 +1,397 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "version": "2020-02-02-preview",
+    "title": "SqlManagementClient",
+    "description": "The Azure SQL Database management API provides a RESTful set of web APIs that interact with Azure SQL Database services to manage your databases. The API enables users to create, retrieve, update, and delete databases, servers, and other entities."
+  },
+  "host": "management.azure.com",
+  "schemes": [
+    "https"
+  ],
+  "consumes": [
+    "application/json"
+  ],
+  "produces": [
+    "application/json"
+  ],
+  "paths": {
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Sql/servers/{serverName}/devOpsAuditingSettings/{devOpsAuditPolicyName}": {
+      "get": {
+        "tags": [
+          "DevOpsAudit"
+        ],
+        "description": "Gets a server's DevOps audit policy.",
+        "operationId": "ServerDevOpsAuditPolicies_Get",
+        "parameters": [
+          {
+            "$ref": "#/parameters/ResourceGroupParameter"
+          },
+          {
+            "$ref": "#/parameters/ServerNameParameter"
+          },
+          {
+            "name": "devOpsAuditPolicyName",
```

Comment:
I don't see a delete, is this a singleton that exist by default and can only be updated? If so we should document the policy name here.

Reply:
yes. That is correct. its exists by default (and default is disabled policy).
how would you suggest documenting this?

Reply:
parameters can have descriptions, please add a description documenting the behavior

Reply:
@harpaul-gill  @ericshape